src/NuGet.Core/NuGet.Configuration/PackageSource/PackageSourceUpdateSettings.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.Configuration
+{
+    public sealed class PackageSourceUpdateSettings
```

Comment:
I don't know if you discussed it with @zivkan, but my suggestion would've been PackageSourceUpdateOptions because Setting is such an overloaded term throughout the NuGet type.

Reply:
lol, I didn't think about that. It's a settings update setting :)